keylime/web_util.py

```diff
@@ -406,14 +415,14 @@ def echo_json_response(handler, code, status=None, results=None):
     return False
 
 
-def get_restful_params(urlstring):
+def get_restful_params(urlstring: str) -> Dict[str, Union[str, None]]:
```

Comment:
Not sure, but maybe `Dict[str, Optional[str]]`

Reply:
right. I did this kind of on purpose in this case to make the `None` visible.